ajax/libs/upng-js/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "upng-js",
+  "description": "Small, fast and advanced PNG / APNG encoder and decoder",
+  "version": "2.0.0",
+  "author": "photopea",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/photopea/UPNG.js"
+  },
+  "filename": "UPNG.min.js",
+  "npmName": "upng-js",
+  "npmFileMap": [
```

Comment:
Let's move `npmName` and `npmFileMap` below all the fields.
Let the fields' order as the usual order to help people can easy to read this file.
Thanks